testutil/integration/evmos/utils/bank_test.go

```diff
@@ -0,0 +1,65 @@
+package utils_test
+
+import (
+	"testing"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	grpchandler "github.com/evmos/evmos/v16/testutil/integration/evmos/grpc"
+	testkeyring "github.com/evmos/evmos/v16/testutil/integration/evmos/keyring"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/network"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheckBalances(t *testing.T) {
+	testDenom := "atest"
```

Comment:
Why is this needed?

Reply:
just to check it also works not only for the basic Evmos denom 👀 not 100% necessary but also doesn't harm